.appveyor.yml

```diff
@@ -14,14 +14,17 @@ build:
 
 environment:
     matrix:
-      - CMAKE_GENERATOR: "Ninja"
-      - CMAKE_GENERATOR: "MinGW Makefiles"
```

Comment:
Also, I think this is the first time we are actually testing MSVC, because Ninja would always pick MinGW and I never noticed it.